python/ray/autoscaler/kubernetes/example-minimal.yaml

```diff
@@ -56,3 +56,26 @@ provider:
             kind: Role
             name: autoscaler
             apiGroup: rbac.authorization.k8s.io
+
```

Comment:
Hmmm these examples seem pretty long still. Do you have any thoughts on what it would take to move more of these into `defaults.yaml`?

Reply:
hmm, fillout_defaults only fills in top-level fields

The examples could be shortened if fillout_defaults filled in the subfields of the provider config that give the head node autoscaler permissions.

Reply:
Is that hard to change?

Reply:
Another thing to consider is that defaults need to be filled out in slightly different ways when using K8s operator vs. when using cluster launcher with K8s.

In any case, I think it's sufficiently complicated [enough decisions involved] that it's better left to another PR.